web-bundle/src/main/java/com/graphhopper/resources/InfoResource.java

```diff
@@ -102,19 +103,17 @@ public Info getInfo() {
                 continue;
 
             List<Object> possibleValueList = new ArrayList<>();
-            try {
-                EnumEncodedValue eev = storage.getEncodingManager().getEncodedValue(encodedValue, EnumEncodedValue.class);
-                for (Object o : eev.getValues()) {
+            EncodedValue eev = storage.getEncodingManager().getEncodedValue(encodedValue, EncodedValue.class);
+            if (eev instanceof EnumEncodedValue) {
+                for (Object o : ((EnumEncodedValue) eev).getValues()) {
                     possibleValueList.add(o.toString());
                 }
-            } catch (ClassCastException e) {
-                try {
-                    BooleanEncodedValue bev = storage.getEncodingManager().getEncodedValue(encodedValue, BooleanEncodedValue.class);
-                    possibleValueList.add(true);
-                    possibleValueList.add(false);
-                } catch (ClassCastException ex) {
-                    continue;
-                }
+            } else if (eev instanceof BooleanEncodedValue) {
+                possibleValueList.add(true);
+                possibleValueList.add(false);
```

Comment:
When look into this again, I think there is something wrong .. either in the spec or here (it changed over the last months ;)). Because we currently have to specify it like:

```
priority:
   get_off_bike: 0.5
```

And not

```
priority:
   get_off_bike: 
      true: 0.5
```

Because specifying both (true and false) makes no sense.

Reply:
> Because specifying both (true and false) makes no sense.

You mean it makes no sense to specify both because only the relative size of the priority counts? For example if we set `{true: 0.6, false: 0.3}` it still means that the priority of edges with `true` is twice as high as the those with `false` (and every edge is either `true` or `false`, right? there is no `none`)? And by the way what if I set the priority to `50` here? (Why) do we not need a normalization as we do for enum encoded values?

Reply:
This is now solved as we use the same procedure like we do for enums. And because we cannot "prefer" a value anymore we should be able to use both, the `true` and the `false` entry.